src/widgets/listbox.cpp

```diff
@@ -319,6 +319,17 @@ namespace gcn
     {
         if (mListModel != NULL)
         {
+            int maxElementLength = getWidth();
```

Comment:
I would avoid mixing length and width, just use width everywhere here.

Reply:
Fixed